src/components/Questionnaire/QuestionnaireEditor.tsx

```
@@ -74,9 +74,15 @@
 
 import mutate from "@/Utils/request/mutate";
 import query from "@/Utils/request/query";
+
+import { PaginatedResponse } from "@/Utils/request/types";
+import { MoveDown, MoveUp } from "@/Utils/request/utils";
+
 import { HTTPError, PaginatedResponse } from "@/Utils/request/types";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'PaginatedResponse'. Consider to delete it or rename it.

'PaginatedResponse' is defined here:


(lint/suspicious/noRedeclare)